src/components/BottomMenu2.js

```diff
@@ -11,9 +11,16 @@ const BottomMenu = ({ navigation }) => {
   // console.log(routeName);
   return (
     <View style={styles.container}>
-      <TouchableOpacity style={styles.menuItem} onPress={() => navigation.navigate('SavedRecipeScreen')}>
-        <Entypo name="open-book" size={24} color="black" />
-        <Text style={styles.menuItemText}>Saved Recipes</Text>
+      <TouchableOpacity style={styles.menuItem} onPress={() => navigation.navigate('RecipeScreen')}>
+        <Entypo
+          name="open-book"
+          size={24}
+          color={routeName === 'RecipeScreen' ? 'white' : 'black'}
+        />
+        <Text>
+          style={routeName === 'RecipeScreen' ? styles.menuItemTextScreen : styles.menuItemText}>
+          Saved Recipes
```

Comment:
`<Text style={routName==....}`
I think the bracket should be open for line 20. But other than that it runs well!